components/PendingCommunityTransactions/PendingCommunityTransactions.tsx

```diff
@@ -0,0 +1,49 @@
+import { Fieldset } from 'components/Fieldset';
+import { ethers } from 'ethers';
+import { PendingChanges } from 'lib/communityNFT/multisig';
+import { convertIPFS } from 'lib/getNFTInfo';
+import Link from 'next/link';
+import styles from './PendingCommunityTransactions.module.css';
+
+type PendingCommunityTransactionsProps = {
+  multiSigChanges: { [key: number]: PendingChanges[] };
+};
+
+export function PendingCommunityTransactions({
+  multiSigChanges,
+}: PendingCommunityTransactionsProps) {
+  return (
+    <div>
+      {Object.keys(multiSigChanges).map((nonce) => (
+        <div className={styles.wrapper} key={nonce}>
+          <Fieldset legend={`Pending Transaction #${nonce}`} key={nonce}>
+            {multiSigChanges[parseInt(nonce)].map((change, i) => {
+              return (
+                <div key={i}>
```

Comment:
nit: using the index in a list as key is not ideal. Looks like each change has a unique id we could use?

Reply:
@cnasc nope, that id is for the community or accessory ID, unfortunately i think this is our only option

i could also append the index to the nonce, and that would be unique as well?

Reply:
nevermind, can use account + change id, and that is unique
cc @cnasc